### ISSUE TITLE
Image creation fails with no meaningful errors

### ISSUE THREAD

**Issue by t0v** [testcontainers/testcontainers-node#89]
i'm trying to build an image with
```
    const buildContext = path.resolve(__dirname, '../../');
    try {
        const container = await GenericContainer.fromDockerfile(buildContext)
        .build();
    }
    catch (e) {
        console.log(e);
    }
```
and that just fails after ~40 seconds w/o giving any reasonable error message:
```
  testcontainers INFO: Using Docker defaults +0ms
  testcontainers INFO: Building image '4e0f7a06234ada9ac05edc4a278b7a09:1fb761b7a169152a983a78ee352eeafb' with context '/Users/me/project' +1ms
Error: Failed to build image
    at GenericContainerBuilder.<anonymous> (/Users/me/node_modules/testcontainers/dist/generic-container.js:44:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/me/node_modules/testcontainers/dist/generic-container.js:4:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

i can see that the building starts, but cannot get logs from the 'build' container
`Error response from daemon: configured logging driver does not support reading`

running node v14.3.0 on macOS 10.13.6 and docker desktop 2.1.0.4 (39773) engine 19.03.4

building the container manually works fine.

is there a way to get logs from the 'build' container?

**Comment by t0v** [comment]
i did some more investigation and found the root issue: build ignores `.dockerignore`.

**Comment by cristianrgreco** [comment]
Fixed in 2.13.0, container output is now TRACE logged with the container ID:

![Screenshot from 2020-06-27 17-31-50](https://user-images.githubusercontent.com/10348798/85927139-19792480-b89c-11ea-8dc3-894666a3a3f6.png)

**Comment by cristianrgreco** [comment]
I will raise a separate issue to address .dockerignore being ignored
